src/App.tsx

```diff
@@ -1,10 +1,23 @@
-import { Counter } from './components';
+import { AppRouter } from './lib/Router/AppRouter';
+import { Home } from './pages';
+import { Login } from './pages';
 
-export const App = () => {
+const routes= [
+  {
+    path: '/',
+    element: <Home />,
+  },
+  {
+    path: '/login',
+    element: <Login />,
+  },
+];
```

Comment:
@ISHIMWE-Aime Create a folder for routes and export routes.

Reply:
Definitely; 
My bad I have forget that.